tests/integration/core/test_status_handler.py

```diff
@@ -95,19 +95,8 @@ def test_save(instance):
         == ACTIVATION_STATUS_MESSAGE_MAP[instance.status]
     )
 
-    for status in [
-        ActivationStatus.STARTING,
-        ActivationStatus.RUNNING,
-        ActivationStatus.STOPPING,
-        ActivationStatus.DELETING,
-        ActivationStatus.COMPLETED,
-        ActivationStatus.FAILED,
-        ActivationStatus.STOPPED,
-        ActivationStatus.UNRESPONSIVE,
-        ActivationStatus.ERROR,
-        ActivationStatus.PENDING,
-    ]:
-        instance.status = status
+    for status in ActivationStatus:
+        instance.status = status.value
```

Comment:
Do we need to call `value` here?

Reply:
probably don't, but it's the interface of every enum, so it's valid.